api/actions/createWorkItem.js

```diff
@@ -67,28 +79,39 @@ module.exports = {
                                 interval: 60000
                             });
                             if (syncedItem) {
-                                return await Github.createComment({
-                                    req,
-                                    body: `This issue has been linked to a new work item: ${getWorkItemUrl(
-                                        syncedItem,
-                                        hideWorkItemUrl
-                                    )}`
-                                });
+                                return await updateIssue(req, `This issue has been linked to a new work item: ${getWorkItemUrl(syncedItem, hideWorkItemUrl)}`, suppressGithubComments);
                             }
-                            return await Github.createComment({
-                                req,
-                                body:
-                                    'Sorry we could wait until Heroku connect make the syncronization.'
-                            });
+                            return await updateIssue(req, 'Sorry we could wait until Heroku connect make the synchronization.', suppressGithubComments);
                         }
                     }
                 );
             }
-            return await Github.createComment({
-                req,
-                body: getBuildErrorMessage(config, milestone)
-            });
+            return await updateIssue(req, getBuildErrorMessage(config, milestone, suppressGithubComments));
         }
         return null;
     }
 };
+function formatToGus(url, body) {
+    var formattedDescription;
+    remark().use(markdown).use(strip).process(body, (err, file) => {
```

Comment:
Strip markdown to GUS format

Reply:
This show how this works and I tried with a few github issues from the past and seems to work as expected - http://remarkjs.github.io/strip-markdown/